.gitignore

```diff
@@ -29,3 +29,9 @@ tmp
 
 # Users Environment Variables
 .lock-wscript
+
+*~
+.DS_Store
+.idea
```

Comment:
It is not necessary to be here. You can setup ignored filed for your text editor in operation system in your home folder. Project's gitignore should contain only files related to project.

See this article: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore